### PR TITLE
Project setup update phase2

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@saeedmb8 @nathanbegbie please review.

File: gulpfile.js [hunk 1]
```diff
@@ -33,18 +33,17 @@ var gulp              =   require('gulp'),
...
     .pipe(sassGlob())
+    .pipe(sass().on('error', sass.logError))
     .pipe(bless())
     .pipe(cleanCSSMinify())
-    .pipe(sassLint())
-    .pipe(sassLint.format())
-    .pipe(sassLint.failOnError())
+    //.pipe(sassLint())
+    //.pipe(sassLint.format())
+    //.pipe(sassLint.failOnError())
```
Comment (nathanbegbie) [inline gulpfile.js]: If we no longer want these lines then let's remove them instead of commenting them out.

**Comment by nathanbegbie** [comment]
@Mitso Could you please give a brief description for this PR? It's a bit difficult to get a sense of what's going on here

**Comment by Mitso** [comment]
@nathanbegbie thanks for the review.

Apply  SASSLINT  _convention 'hyphenatedbem':_  rules, currently not following config rules 

Restructure SASS file imports using Gulp sassGlob, as well as workaround for cascade rules where applicable.
screen shot 2017-10-09 at 5 22 44 pm

Cleanup SASS files, variables, mixins.
Cleanup html pattern files where SASS update is applicable.

File: molo/core/templates/base.html [hunk 2]
```diff
@@ -83,6 +82,7 @@
       {% include "patterns/components/navigation/navigation.html" %}
     {% endblock %}
 
+    <!--Which one is Footer? -->
```
Comment (nathanbegbie) [inline molo/core/templates/base.html]: Random comment?

**Comment by Mitso** [comment]
🍰 🍰 🍰

**Pull request merged**